### PR TITLE
allow 'plugin' extensions to clojure resource server 

### PR DESCRIPTION
**Pull request by loomis**
Connected to #461.

### CODE REVIEW

File: ssclj/rpm/src/main/systemd/ssclj.service [hunk 1]
```diff
@@ -1,13 +1,22 @@
 [Unit]
 Description=SlipStream API
-After=syslog.target network.target
+After=syslog.target
+After=network.target
```
Comment (st) [inline ssclj/rpm/src/main/systemd/ssclj.service]: Is it equivalent to previous version?

Comment (st) [inline ssclj/rpm/src/main/systemd/ssclj.service]: I have another question regarding `create-daily-usage.sh`
should `-cp /opt/slipstream/ssclj/etc` be updated?

Comment (loomis) [inline ssclj/rpm/src/main/systemd/ssclj.service]: Yes, the "After" configurations are equivalent and I think that the one with separated values is clearer.

Comment (loomis) [inline ssclj/rpm/src/main/systemd/ssclj.service]: I'll update the classpath for the script.

**Comment by loomis** [comment]
The classpath for the script was updated to match the new layout of the server.  Please check that it is correct.

**Pull request merged**